experiment_tool/dag_reader.py

```diff
@@ -1,39 +1,51 @@
-from typing import Tuple, Dict, List, Set, Any, Callable
+import importlib
+import logging
+import os
+import sys
+from typing import Callable, Dict, List, Set, Tuple
 
-from .dag import DAG
+from experiment_tool.dag import DAG
 
+log = logging.getLogger(__name__)
 
-def get_first_experiment(experiment_name: str) -> Dict[Callable, List[Any]]:
+
+def get_experiment(experiment_name: str, package: str = 'experiments') \
+        -> Dict[Callable, List]:
     """
-    Get hardcoded function
+    Get hardcoded experiment
     :param experiment_name:
+    :param: package:
     :return: functions with arguments
     """
-    if experiment_name == 'first_example':
-        import experiments.first_example as f
-        return {
-            f.get_5_numbers: [('name',), ('a', 'b', 'c', 's', 'n')],
-            f.slow_1: [('a', 's'), ('id1',)],
-            f.slow_2: [('b', 's'), ('id2',)],
-            f.slow_3: [('c', 's'), ('id3',)],
-            f.fib_1: [('n',), ('x',)],
-            f.fib_2: [('n',), ('y',)],
-            f.fib_3: [('n',), ('z',)],
-            f.get_sum: [('id1', 'id2', 'id3'), ('u',)],
-            f.check: [('x', 'y', 'z'), ('v',)],
-            f.get_result: [('u', 'v'), ('result',)],
-        }
-    raise ValueError('uncorrect experiment name {}'.format(experiment_name))
+    path = sys.path
+    path_to_exp = '{path}/{package}'.format(
```

Comment:
```suggestion
    path_to_exp =  os.path.join(os.getcwd(), package)
```
Этот код генерирует невалидный путь на Windows. 
Правильнее делать так.

Reply:
верно, исправляю